docs/pages/design_tokens.js

```diff
@@ -18,15 +18,60 @@ export type Token = {|
 |};
 
 const tokenCategories = [
-  { name: 'Spacing', category: 'spacing', id: 'space' },
-  { name: 'Background color', category: 'background-color', id: 'background' },
-  { name: 'Text color', category: 'text-color', id: 'color-text' },
-  { name: 'Font size', category: 'font-size', id: 'font-size' },
-  { name: 'Font weight', category: 'font-weight', id: 'font-weight' },
-  { name: 'Font family', category: 'font-family', id: 'font-family' },
-  { name: 'Border color', category: 'color-border', id: 'color-border' },
-  { name: 'Elevation', category: 'elevation', id: 'elevation' },
-  { name: 'Data visualization', category: 'data-visualization', id: 'data-visualization' },
+  {
```

Comment:
This adds a description line to each section to the most closely related component